src/view/Window.cxx

```diff
@@ -82,3 +82,8 @@ void Window::handleEvent(WindowRedrawEvent &&event)
   m_Activity->draw(*m_Renderer);
   m_Renderer->commit();
 }
+
+MouseState & Window::getMouseState() noexcept
+{
+  return m_MouseState;
+}
```

Comment:
Exposing class private filed like that is bad code design.

Reply:
Where would you put this instead?

Reply:
We agreed to pass all global models through ActivityEvent.

Reply:
Targetted to Pt II. We will create a tuple of const references that will provided to iActivity constructor